http.go

```diff
@@ -151,45 +229,71 @@ func (s *httpServer) reloadHandler(w http.ResponseWriter, req *http.Request) {
 }
 
 type statsResponse struct {
-	Requests     uint64        `json:"total_requests"`
-	SeekCount    uint64        `json:"total_seeks"`
-	GetRequests  uint64        `json:"get_requests"`
-	GetHits      uint64        `json:"get_hits"`
-	GetMisses    uint64        `json:"get_misses"`
-	GetAvg       time.Duration `json:"get_average_request"` // Microsecond
-	Get95        time.Duration `json:"get_95"`              // Microsecond
-	Get99        time.Duration `json:"get_99"`              // Microsecond
-	MgetRequests uint64        `json:"mget_requests"`
-	MgetHits     uint64        `json:"mget_hits"`
-	MgetMisses   uint64        `json:"mget_misses"`
-	MgetAvg      time.Duration `json:"mget_average_request"` // Microsecond
-	Mget95       time.Duration `json:"mget_95"`              // Microsecond
-	Mget99       time.Duration `json:"mget_99"`              // Microsecond
-	DBSize       int64         `json:"db_size"`
-	DBMtime      int64         `json:"db_mtime"`
+	Requests        uint64        `json:"total_requests"`
```

Comment:
Might seem silly, but imo formatting changes should be handled separately. I know go formatting does this automatically, but the likelihood of a PR landing upstream decreases with the number of lines it touches. Up to you if you think it's worth reversing the formatting changes though.